### ISSUE TITLE
Loading plugin can not be hidden

### ISSUE THREAD

**Issue by aiwarzheng** [quasarframework/quasar#5100]
Hi Team

I show a loading with Loading plugin, but I can not hide it, quasar version is 1.1.2, but it works fine with version 1.1.0, my code is :

```js
import { Loading, QSpinnerTail } from 'quasar'

let loading = {}
let lastRequest = new Date(2019);

loading.show = function(config) {
    let now = new Date();
    let ms = now - lastRequest;
    lastRequest = now;
    if (ms > 2000) {
        Loading.show({
            spinner: QSpinnerTail,
            message: '',
            messageColor: 'white',
            spinnerSize: 100,
            spinnerColor: 'white',
            customClass: ''
        })
    }
}

loading.hide = function() {
    setTimeout(() => {
        if (Loading.isActive) {
            Loading.hide()
        }
    }, 1000)
}

export default loading;
```

Please advise

Thanks

**Comment by pdanpdan** [comment]
Please try to do a codepen showing the problem. Thank you.

**Comment by rstoenescu** [comment]
Hi,

Please reopen with a codepen or at least the sequence and timing of calls to loading.show() and loading.hide(), otherwise it is unclear what happens there so we cannot help you.

We actually fixed a Loading bug (https://github.com/quasarframework/quasar/issues/4992) in 1.1.0 and battle-tested this a lot. Fixing this might have uncovered a race-condition in your code.
So help us understand your exact scenario please.

Also, it is not necessary to check if Loading is active before hiding it.

Bottom line, please reopen with the requested information. Thank you.

**Comment by kpwbo** [comment]
I have the same kind of bug with 1.1.2. The bug doesn't appear with 1.1.0. I'm not able to reproduce the issue in a codepen. I have explored what happens though when the bug appears.

When `hide()` is called in Loading.js, `this.isActive` is set to false, but the QLoading vm is not re-rendered, the after-leave callback is not called, and the QLoading vm is not destroyed. If I insert `vm.$forceUpdate()` after `this.setActive = false` in the hide method, the vm is destroyed properly and the loader disappears.

**Comment by kpwbo** [comment]
Found the problem! "Loading" was not specified in the plugins section of quasar.conf.js. Apparently that didn't stop it from working (afaik) in 1.1.0, but it does in 1.1.2. Simply adding "Loading" in the plugins list fixed the issue for me. @aiwarzheng is this your problem too?

**Comment by rstoenescu** [comment]
@kpwbo now I see. YES, please do specify any Quasar plugins that you are using in quasar.conf.js. They have initialization code which needs to run before the usage, and adding them to quasar.conf.js is exactly what this does.

**Comment by aiwarzheng** [comment]
@kpwbo  Yes, the problem was solved when added "Loading" in the plugins list, thanks guys

**Comment by Phunky** [comment]
It maybe worth adding this to the docs or at least noting this on the changelog as it's a breaking change that can be unexpected. I just hit this myself while upgrading from `1.0.5` to `1.1.6`.

**Comment by pdanpdan** [comment]
It's not a breaking change according to the existing documentation - in the docs it was always required to add the plugin in the conf. The only strange thing is that it worked before without adding it (and I would expect it not to work as intended).

**Comment by Phunky** [comment]
> It's not a breaking change according to the existing documentation
What broke hasn't been touched for over a year and had been running fine prior to upgrading to the latest version.

I totally understand that this isn't an intentional breaking change, but that doesn't mean it isn't one and that users won't stumble on it. 

An additional note in the changelog would be helpful for those that may stumble on this from legacy usage as we have.
